<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormInfoProcess</class>
 <widget class="QWidget" name="FormInfoProcess">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>238</width>
    <height>142</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Process</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="2">
    <widget class="QToolButton" name="btnCommands">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../../res/resource.qrc">
       <normaloff>:/images/menu.png</normaloff>:/images/menu.png</iconset>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="txtModuleName">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="lblModuleName">
     <property name="text">
      <string>Module name:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lblModulePath">
     <property name="text">
      <string>Module path:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="lblProcessId">
     <property name="text">
      <string>Process Id:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QLineEdit" name="txtModulePath">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="3">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="1" colspan="2">
    <widget class="QLineEdit" name="txtProcessId">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="lblThreadId">
     <property name="text">
      <string>Thread Id:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="QLineEdit" name="txtThreadId">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../res/resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btnCommands</sender>
   <signal>clicked()</signal>
   <receiver>FormInfoProcess</receiver>
   <slot>slotCommands()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>355</x>
     <y>18</y>
    </hint>
    <hint type="destinationlabel">
     <x>188</x>
     <y>128</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slotCommands()</slot>
 </slots>
</ui>
